src/widgets/RRule.tsx

```diff
@@ -0,0 +1,284 @@
+import * as React from 'react';
+import Container from './Container';
+import { TextField, Select, MenuItem, FormGroup, FormControlLabel, Checkbox, InputLabel, FormControl } from '@material-ui/core';
+import RRule, { Options, Weekday, Frequency, ByWeekday } from 'rrule';
+import { ALL_WEEKDAYS } from 'rrule/dist/esm/src/weekday';
```

Comment:
You don't want to import like this. If it's not exposed using a recommended path it probably means it's internal and shouldn't be imported...